regress/pesterTests/SSH.Tests.ps1

```diff
@@ -1,78 +1,56 @@
 ﻿If ($PSVersiontable.PSVersion.Major -le 2) {$PSScriptRoot = Split-Path -Parent $MyInvocation.MyCommand.Path}
-Import-Module $PSScriptRoot\CommonUtils.psm1 -Force
+. $PSScriptRoot\TestUtils.ps1
```

Comment:
I would probably call this common.ps1, since it would also have execution logic apart from utility functions. 

Can common.ps1 take parameters, if so, I would support the following:
- OpenSSHBinPath - for testing an explicit version
- TestDir - for overloading test directory. Default for this could be in users %TEMP% directory